README.md

```diff
@@ -2,222 +2,179 @@
   * Motive: All the software being developed shall have not only unit tests but integration tests, so robots and structures shall be developed for this purpose. These shall include vcstool for specifying packages to be used with the robot as well as ros_ign_bridge to test it in simulation.
-  * Repositories
+  * Proposal
     * **[Proposal]** Hover type UW vehicle
     * **[Proposal]** Torpedo shaped UW vehicle
     * **[Proposal]** USV
-    * **[Proposal]** Intervention UW vehicle 
-  * References: 
+    * **[Proposal]** Intervention UW vehicle
+  * References
     * Intervention UW sample [RexROV2](https://uuvsimulator.github.io/packages/rexrov2/intro/)
-    * Torpedo Shaped UW vehicle [LRAUV](https://github.com/osrf/lrauv/)
+    * Torpedo Shaped UW vehicle [MBARI LRAUV](https://github.com/osrf/lrauv/)
 
 * Simulation
   * Description: Documentation and index for maritime robotics Gazebo plugins.
-  * Motive: There is no need to mix ROS code with simulation, also there is already some maritime robotics development in Ignition, so it is a good thing to contribute to ign-gazebo repo instead of creating other in this organization.
-  * Repositories
+  * Motive: There is no need to mix ROS code with simulation, also there is already some maritime robotics development in new Gazebo, so it is a good thing to contribute to ign-gazebo repo instead of creating other in this organization.
+  * Proposal
     * **[Proposal]** Maritime Robotics Gazebo Plugins Index
-    * **[Proposal]** Wave Simulation (repo shall be in ign-plugins)
-    * **[Proposal]** Acoustics Simulation (repo shall be in ign-plugins) (Include salinity and temperature effects)
-    * **[Proposal]** Radar Simulation (repo shall be in ign-plugins)
-    * **[Proposal]** Sonar Simulation (repo shall be in ign-plugins)
-  * References: 
+    * **[Proposal]** Wave Simulation
+    * **[Proposal]** Acoustics Simulation (include salinity and temperature effects)
+    * **[Proposal]** Radar Simulation
+    * **[Proposal]** Sonar Simulation
+  * References
     * General
       * [Gazebo-classic](https://classic.gazebosim.org/)
-      * [Gazebo Sim (formerly Ignition)](https://github.com/gazebosim/gz-sim)
+      * [Gazebo (formerly Ignition)](https://github.com/gazebosim/gz-sim)
     * Surface
       * [ASV Wave Simulator](https://github.com/srmainwaring/asv_wave_sim/tree/feature/fft_waves)
-      * [Virtual RobotX (VRX)](https://github.com/osrf/vrx): ROS and Gazebo
+      * [Virtual RobotX (VRX) competition](https://github.com/osrf/vrx): ROS and Gazebo-classic / new Gazebo
+      * [MBZIRC Maritime Grand Challenge Simulator](https://github.com/osrf/mbzirc): ROS 2 and Gazebo
+      * [MBARI Wave Energy Converter Simulator](https://github.com/osrf/mbari_wec): ROS 2 and Gazebo
+      * Sailboat and wind simulation of [Racing Sparrow 750](https://github.com/srmainwaring/rs750) and [Docker files](https://github.com/srmainwaring/sail_sim_docker)
```

Comment:
Remove the Docker reference for sail_sim_docker, it's for a previous version that runs on Gazebo Classic.